### ISSUE TITLE
Azure functions extension won't deploy apps

### ISSUE THREAD

**Issue by johnmeister** [microsoft/vscode-azurefunctions#3097]
Does this occur consistently? Yes
Repro steps:

1. select Azure Function from Azure Extension in VS Code.
2. right click select deploy to Function App...
3. Confirm with Deployment popup
4. open bottom panel and watch output window
5. see log outputs below:

"11:26:42 AM infoaxis-website-Dev: Starting deployment...
11:26:49 AM infoaxis-website-Dev: Creating zip package...
11:26:50 AM infoaxis-website-Dev: Zip package size: 36.5 kB
11:29:50 AM: Error: 400.0 BadRequest"

Action: azureFunctions.deploySlot
Error type: 400
Error Message: redacted:id BadRequest

Version: 1.6.1
OS: win32
OS Release: 10.0.22000
Product: Visual Studio Code
Product Version: 1.66.0
Language: en

<details>
<summary>Call Stack</summary>

```
new RestError extension.bundle.js:2:1948514
extension.bundle.js:2:1935988extension.bundle.js:2:1935988
```

</details>

I have attempted on different azure functions, and different deployment slots with the same effect.

**Comment by johnmeister** [comment]
downgraded VS Code to 1.65.2 and it worked

**Comment by klodha** [comment]
Duplicate of #3094 and #2844

**Comment by alexweininger** [comment]
Until a fix is released in VS Code stable (hopefully the next major release), here are some options:
1. Wait for the fix to come in [VS Code Insiders](https://code.visualstudio.com/insiders/)
2. [Deploy via Azure CLI](https://github.com/microsoft/vscode-azurefunctions/issues/2844#issuecomment-1086259916)
3. Downgrade your Azure Functions extension version to `1.2.1` (Feb 08, 2021), we used a different zip deploy method which seems to be working still. Note: it might not work in environments like WSL or Codespaces, which is why we changed our deploy in `1.3.0` (Mar 15, 2021).
4. Downgrade to the [VS Code February 2022 release](https://code.visualstudio.com/updates/v1_65), where zip deploy is working as discussed and confirmed above.
---

Here's some information on the bug we suspect is causing the issue with zip deploy.

We suspect that it was a regression in the [libuv library](https://github.com/libuv/libuv) caused by [libuv/libuv@ce15b84](https://github.com/libuv/libuv/commit/ce15b8405e9d01e221f8390475deab4a40d26e38), which was included in Node.JS `16.13.0`.

See [this issue](https://github.com/microsoft/vscode/issues/146470), and specifically [this comment](https://github.com/microsoft/vscode/issues/146470#issuecomment-1087175293) for more information.

**Comment by alexweininger** [comment]
Closing as a duplicate of #2844

**Comment by alexweininger** [comment]
Fix has been released in version [v1.6.2](https://github.com/microsoft/vscode-azurefunctions/releases/tag/v1.6.2) of the extension.
